### ISSUE TITLE
Keyboard focus order is incorrect inside the ""Available Authorization" dialog when we navigate using TAB key. :A11y_VS Code extension for API Center_Definition_OpenAPIDocumentation_Keyboard. 

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#120]
### GitHub Tags
#A11yTCS; #A11ySev3; #Mac; #WCAG2.4.3; #GH_VSCodeextensionforAPICenter_Mac_Apr2024; #DesktopApp; #VS Code Extension for API Center; #FocusOrder; #AILimited; #A11yMAS; #device; #Keyboard;

### Environment Details:
Application Name: VS Code Extension for API Center
macos sonoma version:14.4.1

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Open API Documentation" using the down arrow and press enter.
7. Demo conference API screen will appear. TAB to "Authorize" button and hit ENTER key.
8. "Available Authorization" dialog will open. Navigate using TAB key and observe the issue.

### Actual Result:
Keyboard focus order is incorrect inside the ""Available Authorization" dialog when we navigate using TAB key.

### Expected Result:
Keyboard focus order should be correct inside the ""Available Authorization" dialog when we navigate using TAB key. As soon as the dialog open, keyboard focus should start from first interactive control.

User Impact:
Keyboard users will face difficulty here as they will have to use shift+ TAB keys to navigate to previous control present in the dialog.

### Attachment:
![052](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/ca316822-eefe-4fa7-a53b-27652f6fca67)
